Firmware/Marlin_main.cpp

```diff
@@ -5561,7 +5561,7 @@ if(eSoundMode!=e_SOUND_MODE_SILENT)
 		eeprom_update_byte((unsigned char *)EEPROM_FARM_MODE, farm_mode);
 		EEPROM_save_B(EEPROM_FARM_NUMBER, &farm_no);
           SilentModeMenu = SILENT_MODE_OFF;
-          eeprom_update_byte((unsigned char *)EEPROM_SILENT, SilentModeMenu);
+          eeprom_update_byte_notify((unsigned char *)EEPROM_SILENT, SilentModeMenu);
```

Comment:
Not sure that you can change this during print

Reply:
You can change Stealth<->Normal mode during printing (even though due to TMC behavior the print is temporarily stopped for a second and resumed after)